README.md

```diff
@@ -42,4 +42,14 @@ docker run \
 sbt <project>/run --help
 ```
 
-e.g. `sbt resourceValidator/run --dryRun --all`
\ No newline at end of file
+e.g. `sbt resourceValidator/run --dryRun --all`
+
+## Contributing
+
+Currently, `com.broadinstitute.dsp.zombieMonitor.DbReaderSpec` and `com.broadinstitute.dsp.resourceValidator.DbReaderSpec` are not run in CI, hence make sure you run them manually before merging any PRs.
+
+Once a PR is merged, there will be a PR created in [terra-helm](https://github.com/broadinstitute/terra-helm) (WIP). 
+Get this PR merged, and another automatic PR will bump leonardo's chart version. This will trigger another automatic PR 
+in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile). Once all these PRs are merged, 
+go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications), and click `sync apps` button for all leonardo deploys (select `PRUNE` option as well),
```

Comment:
I don't see a `sync apps` button
```suggestion
go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications), and click the `Sync` button on the top menu for all leonardo env deploys (select `PRUNE` option as well),
```

Reply:
so there's `SYNC APPS` button on the top left corner..I can add both, but was trying to be more concise and not to confuse ppl

Reply:
I can't see a `SYCN APPS` button on https://ap-argocd.dsp-devops.broadinstitute.org/applications/leonardo-dev. Am I looking at a wrong place?

Reply:
I see...it's at https://ap-argocd.dsp-devops.broadinstitute.org/applications